control-plane/go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/hashicorp/consul-k8s/control-plane/cni v0.0.0-20230825213844-4ea04860c5ed
 	github.com/hashicorp/consul-server-connection-manager v0.1.6
 	github.com/hashicorp/consul/api v1.26.1
-	github.com/hashicorp/consul/proto-public v0.5.1
+	github.com/hashicorp/consul/proto-public v0.1.2-0.20240110225000-635cdcf6e070
```

Comment:
Once you've updated the `replace` pin like you did above, you can just drop this change